editor/modes/visual-editor/block.js

```diff
@@ -252,7 +284,18 @@ class VisualEditorBlock extends Component {
 	onKeyDown( event ) {
 		const { keyCode, target } = event;
 
-		this.handleArrowKey( event );
+		//this.handleArrowKey( event );
```

Comment:
Yep, I should probably remove the function it references too. I am still considering if it contains something I needed to implement. I think I may have removed arrow key navigation between blocks which is something that probably needs to be added back.